src/aktualizr_primary/secondary.cc

```diff
@@ -51,8 +51,8 @@ void initSecondaries(Aktualizr& aktualizr, const boost::filesystem::path& config
       Secondaries secondaries = createSecondaries(*config);
 
       for (const auto& secondary : secondaries) {
-        LOG_INFO << "Adding Secondary to Aktualizr."
-                 << "HW_ID: " << secondary->getHwId() << " Serial: " << secondary->getSerial();
+        LOG_INFO << "Adding Secondary with ECU serial: " << secondary->getSerial()
```

Comment:
mixing metadata/values with message text might not be optimal as you need to search for values between text. The best practice here is to separate a static part of a log message from its variable part. For example,
`
Adding Secondary:
    serial: <serial>
    hardware ID: <hwid>
`

but it's not critical here as there are just two variable elements here, it's rather important when there are more of them

Reply:
I'm not quite sure I follow. All I did was rephrase it and swap the serial and hardware ID.